hibp.js

```diff
@@ -142,7 +148,12 @@ const HIBP = {
 
 
   getBreachByName(allBreaches, breachName) {
-    return allBreaches.find(breach => breach.Name.toLowerCase() === breachName.toLowerCase());
+    breachName = breachName.toLowerCase();
+    if (RENAMED_BREACHES.includes(breachName)) {
```

Comment:
Actually, stupid nit but I'll call it out before I forget and we can address in a future PR if you want me to file a proper bug...

Instead of both `RENAMED_BREACHES[]` and `RENAMED_BREACHES_MAP{}`, what if this just did something like:

```js
if (Object.keys(RENAMED_BREACHES_MAP).includes(breachName)) {...}`
```

By using `Object.keys()`, I _think_ we can get rid of the `RENAMED_BREACHES` array const and only have one place where we define mappings (instead of both the array and object).

🤷